src/main/java/io/javalin/Javalin.java

```diff
@@ -375,6 +372,15 @@ public Javalin requestLogger(@NotNull RequestLogger requestLogger) {
         return this;
     }
 
+    public Javalin enableMetrics() {
+        ensureActionIsPerformedBeforeServerStart("Enable application metrics");
+        Util.INSTANCE.ensureDependencyPresent(OptionalDependency.MICROMETER);
+        final StatisticsHandler statisticsHandler = new StatisticsHandler();
+        this.jettyServer.insertHandler(statisticsHandler);
```

Comment:
@tipsy Not sure if it is the best approach to overwrite the handler you set in [JettyServerUtil](https://github.com/tipsy/javalin/blob/d7ba1f12671ae5c1c51aacc7952b348ea7e685f4/src/main/java/io/javalin/core/util/JettyServerUtil.kt#L35) but might be the most simple one.